docs/model/eland_requirements.rst

```diff
@@ -85,26 +85,26 @@ FUN-MVS-04 - Optimisation Results
 The results of the MVS simulation are post-processed, and result in numberous key performance indicators (KPI). Following information is calculated for an economic evaluation of the energy system:
 
 * Capital and operational expenditures (capex, opex) per asset, both as annuities as well as present costs. This includes also the first-time investment costs (FIC), the replacement costs minus residual values, and the costs for asset dispatch (equations compare :ref:`economic_precalculation-label`).
-* :ref:`NPC <net_present_cost>` and annuity of the whole energy system
-* :ref:`Levelized cost of energy (LCOE) <lcoe>` of the energy system, in electricity equivalent
-* :ref:`Levelized cost of an energy carrier <lcoe>` in electricity equivalent (LCOEleq) for each energy carrier in the energy system
-* :ref:`Levelized cost of asset dispatch <lcoe_asset>`, calculated from the annuity of an asset and their throughput
+* :ref:`NPC <costs_total>` and annuity of the whole energy system
```

Comment:
Yes, the name in the `.csv` for `costs_total` is also `NPC` (https://multi-vector-simulator.readthedocs.io/en/latest/model/simulation_outputs.html#net-present-costs-npc-costs-total) we could also use `Net present costs` here, though.

At some point we should actually change `costs_total` into `NPC` directly in the code, so that it is also displayed correctly in the report. Multiple ways to do this (either translate for the report/outputs only, or translate when adding the keys to the `dict_values`)